### PR TITLE
Circle ci

### PR DESCRIPTION
**Pull request by imnutz**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  build:
+    description: "Building project..."
+    steps:
+      - run: yarn build
+  test:
+    description: "Testing with BrowserStack..."
+    steps:
+      - run: yarn test-full
+
+workflows:
```
Comment (tung-vu-td) [inline .circleci/config.yml:25]: Maybe send workflows to bottom of files similar to other mobile sdk? :p
IMO, it's more naturally read: commands -> jobs -> workflows

**Review by tung-vu-td — APPROVED** [review]
LGTM

**Review by flexgrip — APPROVED** [review]
Reviewed for data exfiltrations.

**Review by nukala — APPROVED** [review]
Reviewed for Data Exfiltrations

**Pull request merged**